ross/fluid_flow/fluid_flow.py

```diff
@@ -0,0 +1,400 @@
+                        plt.plot(0, 0, '*')
+                        plt.title('Cut in plane Z=0')
+                        plt.xlabel('X axis')
+                        plt.ylabel('Y axis')
+                        plt.axis('equal')
+
+                self.re[i][j] = radius_external
+
+                self.ri[i][j] = radius_internal
+        if plot_cut:
+            plt.show()
+            plt.close('all')
+
+    def internal_radius_function(self, z, gama):
+        """This function calculates the radius of the rotor given the
+        radius and the position z.
+        Parameters
+        ----------
+        z: float
+            Distance along the z-axis.
+        gama: float
+            Gama is the distance in the theta-axis. It should range from 0 to 2*np.pi.
+        """
+
+        e = np.sqrt(self.xi ** 2 + self.yi ** 2)
+        if self.xi > 0:
+            beta = np.arctan(self.yi / self.xi)
+        elif self.xi < 0:
+            beta = -np.pi + np.arctan(self.yi / self.xi)
+        else:
+            if self.yi > 0.:
+                beta = np.pi / 2.
+            else:
+                beta = -np.pi / 2.
+        alpha = gama - beta
+        radius_external = e * np.cos(alpha) + np.sqrt(
+            self.radius_rotor ** 2 - (e * np.sin(alpha)) ** 2
+            )
+        xre = radius_external * np.cos(gama)
+        yre = radius_external * np.sin(gama)
+        return radius_external, xre, yre
+
+    def external_radius_function(self, gama):
+        """This function calculates the radius of the stator.
+        Parameters
+        ----------
+        gama: float
+            Gama is the distance in the theta-axis. It should range from 0 to 2*np.pi.
+        """
+
+        radius_internal = self.radius_stator
+        xri = radius_internal * np.cos(gama)
+        yri = radius_internal * np.sin(gama)
+
+        return radius_internal, xri, yri
+
+    def plot_pressure_z(self, show_immediately=True):
+        """This function assembles pressure graphic along the z-axis.
+        Parameters
+        ----------
+        show_immediately: bool
+            If True, immediately plots the graphic. Otherwise, the user should call plt.show()
+            at some point. It is useful in case the user wants to see one graphic alongside another.
+        """
+        if not self.pressure_matrix_available:
+            sys.exit('Must calculate the pressure matrix.'
+                     'Try calling calculate_pressure_matrix first.')
+        plt.figure(self.plot_counter)
+        self.plot_counter += 1
+        for i in range(0, self.nz):
+            plt.plot(i*self.dz, self.p_mat[i][0], 'bo')
+        plt.title('Pressure along the Z direction (direction of flow); Theta=0')
+        plt.xlabel('Points along Z')
+        plt.ylabel('Pressure')
+        plt.show(block=show_immediately)
+        if show_immediately:
+            plt.close('all')
+
+    def plot_shape(self, theta=0, show_immediately=True):
```

Comment:
When using a function to create a plot with matplotlib, it is better to make the axes an argument:
```python
def foo(ax=None):
    # if axes is not given get the one from the current figure
    if ax is None:
        ax = plt.gca()
    .
    .
    ax.plot(...)
    .
    .
    return ax
```
For more details about matplotlib objects (figure, axes, axis etc.) refer to https://matplotlib.org/tutorials/introductory/usage.html

Reply:
We will soon change from matplotlib to bokeh, so I think we can let them like this for now. What do you think?